src/cryptoadvance/specter/templates/includes/address-row.html

```diff
@@ -0,0 +1,75 @@
+<template id="address-row">
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='styles.css') }}">
+    <tr class="address-row">
+        <td class="index"></td>
+        <td class="address"></td>
+        <td class="label"></td>
+        <td class="used"></td>
+        <td class="utxo"></td>
+        <td><span class="amount"></span> <span class="amount-price note hidden">()</span></td>
+        <td></td>
+    </tr>
+</template>
+
+<script type="text/javascript">
+    class TxRowElement extends HTMLElement {
+        constructor() {
+            super();
+            var shadow = this.attachShadow({ mode: 'open' });
+            var style = document.getElementById('address-row').content;
+            var clone = style.cloneNode(true);
+            this.el = clone.querySelector(".address-row");
+
+            this.index = clone.querySelector(".index");
+            this.address = clone.querySelector(".address");
+            this.label = clone.querySelector(".label");
+            this.used = clone.querySelector(".used");
+            this.utxo = clone.querySelector(".utxo");
+            this.amountText = clone.querySelector(".amount");
+            this.amountPrice = clone.querySelector(".amount-price");
+
+            // Attach the created element to the shadow dom
+            shadow.appendChild(clone);
+        }
+
+        connectedCallback() {
```

Comment:
What is this callback doing? I'm not sure how IDE's are supporting javascipt-development and i'm not a good javascript-developer myself that's why documenting javascript code is, at least for me, especially helpfull.

But in general, documenting code is helpful anyway because otherwise you need to dig through the implementation of the code to understand what stuff is doing. Might help to understand the whole codebase but it's scaling very well.

Code is usually once written but read 10+.

Reply:
I am not sure if it should be documented, since it is a function related to lifecycle of the custom HTML elements, not to a specific logic of Specter Wallet.

Anyway, I've added a documentation that describe the purpose of this function.